Applications/Spire/Source/BookView/BookViewLevelPropertiesWidget.cpp

```diff
@@ -131,8 +131,9 @@ BookViewLevelPropertiesWidget::BookViewLevelPropertiesWidget(
   band_properties_layout->addStretch(10);
   auto apply_gradient_button = new FlatButton(tr("Apply Gradient"), this);
   apply_gradient_button->setFixedSize(BUTTON_SIZE());
-  apply_gradient_button->connect_pressed_signal(
-    [=] { on_gradient_apply_button_clicked(); });
+  connect(apply_gradient_button, &FlatButton::clicked, [=] {
```

Comment:
@kranar I can't remember the details, but there was an instance where you preferred using a lambda as the receiver function. Is that always the case, or in cases like this should the signal call on_gradient_apply_button_clicked on 'this'?

Reply:
Use a closure only if you need to capture more than ```this```.